django_redshift_backend/base.py

```diff
@@ -92,8 +92,9 @@ def distinct_sql(self, fields, *args):
         if fields:
             # https://github.com/jazzband/django-redshift-backend/issues/14
             # Redshift doesn't support DISTINCT ON
-            raise NotSupportedError('DISTINCT ON fields is not supported '
-                                      'by this database backend')
+            raise NotSupportedError(
+                'DISTINCT ON fields is not supported by this database backend'
+            )
```

Comment:
This [was failing flake8](https://github.com/browniebroke/django-redshift-backend/runs/1364132287?check_suite_focus=true#step:7:13).